Gemfile.lock

```diff
@@ -1,75 +1,75 @@
 GIT
   remote: https://github.com/sgruhier/foundation_rails_helper.git
-  revision: 45709e28f567b2e619a06091eb63d4f050f861d8
+  revision: bc33600db7a2d16ce3cdc1f8369d0d7e7c4245b5
   specs:
-    foundation_rails_helper (3.0.0)
-      actionpack (>= 4.1, < 7.0)
-      activemodel (>= 4.1, < 7.0)
-      activesupport (>= 4.1, < 7.0)
-      railties (>= 4.1, < 7.0)
+    foundation_rails_helper (4.0.1)
+      actionpack (>= 4.1, < 7.1)
+      activemodel (>= 4.1, < 7.1)
+      activesupport (>= 4.1, < 7.1)
+      railties (>= 4.1, < 7.1)
 
 PATH
   remote: .
   specs:
     decidim-phone_authorization_handler (1.0.0)
-      decidim-core (= 0.25.2)
+      decidim-core (= 0.26)
```

Comment:
@eliegaboriau Is there a way to make it compatible with all 0.26 versions (0.26.1, 0.26.2, etc) ? It causes bundle errors

Reply:
Should be okay now !